requirements.txt

```diff
@@ -20,4 +20,5 @@ selenium==4.2.0
 altair_saver
 webdriver-manager
 s3fs
-https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.1.0/en_core_web_sm-3.1.0.tar.gz
\ No newline at end of file
+https://github.com/explosion/spacy-models/releases/download/en_core_web_sm-3.1.0/en_core_web_sm-3.1.0.tar.gz
```

Comment:
think this will be a separate PR but we should triple check all spaCy requirements given @Jack-Vines 's doozy of a time with it